fe/src/pages/Login/Login.module.scss

```diff
@@ -1,19 +1,3 @@
-.input {
-    margin-bottom: 20px;
-    color: black;
-    height: 30px;
-    width: 250px;
-}
-.h1 {
-    font-size: 30px;
-    margin-bottom: 20px;
-}
-.button {
-    background-color: white;
-    color:black;
-    width: 75px;
-    height: 30px;
-}
 .login {
```

Comment:
Actually you don't need any custom style at all for this element.
Take a closer look at the chakra ui doc [here](https://chakra-ui.com/docs/layout/flex)